src/Mono.Android/Android.Runtime/DynamicMethodNameCounter.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading;
+
+namespace Android.Runtime {
+	internal class DynamicMethodNameCounter {
+		static int dynamicMethodNameCounter;
+
+		internal static string UniqueName
```

Comment:
This doesn't match our normal formatting guidelines; `{` should be on the same line as the property name.

However, this should be a property; it should be a method.  "Normal" property semantics are that the getter won't return a different value every time it's called.  (Yes, `DateTime.Now` doesn't do that.  Yes, it should never have been a property in the first place.  We shouldn't repeat the design mistakes of `DateTime.Now`.)

Since this *will* return a new value every time it's called, it should be a method.

Reply:
That is one of the conventions, which doesn't make much sense to me :-)